src/site/layouts/q_a.drupal.liquid

```diff
@@ -70,6 +70,7 @@
       </div>
     </div>
   </main>
+  {% include "src/site/components/up_to_top_button.html" %}
```

Comment:
@zacharymorel I noticed that the FAQ Multiple layout includes several of these buttons where there is a heading.

```
<button type="button" class="vads-u-visibility--screen-reader">Back to top</button>
```

I'm out of the loop on the convo behind that. Do we need to add those buttons into each of these other layouts too?

Reply:
I'm also out of the loop, @youngfreezyVA would know more I think?